projects/1.20/assets/everythingcopper/everythingcopper/lang/zh_cn.json

```
@@ -0,0 +1,365 @@
+{
+    "entity.everythingcopper.copper_golem": "铜傀儡",
+    "entity.everythingcopper.unaffected_copper_golem": "铜傀儡",
+    "entity.everythingcopper.exposed_copper_golem": "斑驳的铜傀儡",
+    "entity.everythingcopper.weathered_copper_golem": "锈蚀的铜傀儡",
+    "entity.everythingcopper.oxidized_copper_golem": "氧化的铜傀儡",
+    "entity.everythingcopper.waxed_unaffected_copper_golem": "涂蜡铜傀儡",
+    "entity.everythingcopper.waxed_exposed_copper_golem": "斑驳的涂蜡铜傀儡",
+    "entity.everythingcopper.waxed_weathered_copper_golem": "锈蚀的涂蜡铜傀儡",
+    "entity.everythingcopper.waxed_oxidized_copper_golem": "氧化的涂蜡铜傀儡",
```

```suggestion
    "entity.everythingcopper.waxed_unaffected_copper_golem": "涂蜡的铜傀儡",
    "entity.everythingcopper.waxed_exposed_copper_golem": "涂蜡的斑驳铜傀儡",
    "entity.everythingcopper.waxed_weathered_copper_golem": "涂蜡的锈蚀铜傀儡",
    "entity.everythingcopper.waxed_oxidized_copper_golem": "涂蜡的氧化铜傀儡",
```
1.19.3及以上版本原版翻译换了顺序，请注意同时修改1.20和1.21两个版本的文件